salt/kubelet/stop.sls

```diff
@@ -11,7 +11,7 @@ include:
 drain-kubelet:
   cmd.run:
     - name: |
-        kubectl --kubeconfig={{ pillar['paths']['kubeconfig'] }} drain {{ grains['nodename'] }} --ignore-daemonsets --grace-period=300 --timeout=340s
+        kubectl --kubeconfig={{ pillar['paths']['kubeconfig'] }} drain {{ grains['nodename'] }} --force --delete-local-data=true --ignore-daemonsets --grace-period=300 --timeout=340s
     - check_cmd:
       - /bin/true
```

Comment:
Do we need the `check_cmd: /bin/true` If the `drain` is going to be forced?

Reply:
Yes, it can still have a retvalue != 0 (timeout for example when draining).

Reply:
Yep, we want this to *always* succeed, if we can't drain, it's okay - Kubernetes will do the right thing for us once it boots back up.